JetNews/app/src/androidTest/java/com/example/jetnews/JetnewsUiTest.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2019 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.jetnews
+
+import androidx.test.filters.MediumTest
+import androidx.ui.test.assertIsDisplayed
+import androidx.ui.test.createComposeRule
+import androidx.ui.test.doClick
+import androidx.ui.test.findByText
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@MediumTest
+@RunWith(JUnit4::class)
```

Comment:
Does this not need to be `@RunWith(AndroidJUnit4::class)`? If so, why?

Reply:
This is what the Jetpack Compose team uses internally and we wanted to keep the same format for now. They use it because `AndroidJUnit` runner does not work with `ParametrizedTests`.

Also, removing this line was causing a lot of issues when running the tests from Android Studio, giving a `java.lang.AssertionError: The component does not exist!` error.